README.md

```diff
@@ -40,3 +40,12 @@ computers.
 ## Basic use
 
 See [CLI](./cli/README.md).
+
+## Contributing
+
+The [roadmap](https://www.notion.so/pl-strflt/Zinnia-Roadmap-b849d322502f4c079525cd8cc887df02) and [issue tracker](https://github.com/filecoin-station/zinnia/issues) are great places to check out for your first
```

Comment:
I think this is effectively closing the following issue:
- https://github.com/filecoin-station/zinnia/issues/28

Reply:
good catch!